### DISCUSSION TITLE
Description sub-section scroll

### DISCUSSION THREAD

**Discussion by Amorim33** [scalar/scalar#1954] [Ideas]
## Description
[Scalar sub-section.webm](https://github.com/scalar/scalar/assets/42624869/94bd24c9-133f-4444-a135-61193df48b6d)

When adding a H3 (###) to the description section, Scalar creates a sub-section in the left sidebar. However when the item menu is clicked, the page does not scroll to the selected sub-section.

**Comment by Amorim33** [comment]
If it makes sense I will be pleased to work on it at the weekend 😊

**Reply by hanspagel** [reply]
Oh, yes. That’s a bug. We saw that, too and tried to fix it, but it’s way more complex than it seems. 😆 We’re wrapping the sections with an id, that’s way harder with two different levels.

Anyway, I created an issue here: https://github.com/scalar/scalar/issues/2004
